vividus-plugin-rest-api/src/main/java/org/vividus/steps/api/HttpRequestSteps.java

```diff
@@ -77,6 +81,30 @@ public void request(DataWrapper content)
         httpTestContext.putRequestEntity(requestEntity);
     }
 
+    /**
+     * Sets application/x-www-form-urlencoded request entity that will be used while executing request.
+     * HTTP request header with name 'Content-Type' and value 'application/x-www-form-urlencoded; charset=ISO-8859-1'
+     * is set.
+     * <div>Example:</div>
+     * <code>
+     *   <br>Given form data for request body:
+     *   <br>|firstName|lastName|password  |
+     *   <br>|Ivan     |Ivanov  |!@3qwer   |
+     * </code>
+     * <br>
+     * <br>where
+     * @param parameters Any valid ExamplesTable
+     */
+    @Given("urlencoded request: $parameters")
```

Comment:
```suggestion
    @Given("form data request:$formData")
```

Reply:
should not we reflect in step that this is url encoded data?